python_cross_site_scripting/Marv0659_Web_dev_eksamen/x.py

```
@@ -0,0 +1,512 @@
+
+##############################
+def send_verify_email(user_email, user_verification_key):
+    try:
+        # Create a gmail fullflaskdemomail
+        # Enable (turn on) 2 step verification/factor in the google account manager
+        # Visit: https://myaccount.google.com/apppasswords
+
+
+        # Email and password of the sender's Gmail account
+        sender_email = "webdevjenner@gmail.com"
+        password = "crrn qrfi uusx cduj"  # If 2FA is on, use an App Password instead
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "password"

(S105)